rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -507,7 +507,7 @@ class WriterInfo : public ReaderListener
   {
     WriterProxyData proxyData;
     if (change->kind == ALIVE) {
-      CDRMessage_t tempMsg;
+      CDRMessage_t tempMsg(0);
```

Comment:
I don't think the API in FastRTPS needs any changes. More documentation would certainly help. If you think a ticket or patch for FastRTPS makes sense please go ahead and create it.